interface/forms/gad7/new.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/**
+ * gad-7 form using forms api     new.php    create a new form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Ruth Moulton <moulton ruth@muswell.me.uk>
+ * @copyright Copyright (c) 2021 ruth moulton <ruth@muswell.me.uk>
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("gad7.inc.php"); //common strings, other includes etc
+
+use OpenEMR\Common\Csrf\CsrfUtils;    // security module
+use OpenEMR\Core\Header;
+?>
+<html>
+<head>
+<!--   <title><?php echo text($string_form_title); ?> </title> -->
+        <title><?php echo $string_form_title; ?> </title>
+    <?php Header::setupHeader(); ?>
+</head>
+<body class="body_top">
+
+<script>
+var no_qs = 8; // number of questions in the form
+var gad7_score = 0; // total score
+</script>
+
+<SCRIPT
+  src="<?php echo $rootdir;?>/forms/gad7/gad7_javasrc.js">
+ </script>
+
+ <script>
+// stuff that uses embedded php must go here, not in the include javascript file -
+// it must be executed on server side before page is sent to client. included
+// javascript is only executed on the client
+function create_q8(question, menue){
+ // create the 8th question - the second part is italicised. Only displayed if score > 0
+    var text = document.createTextNode(jsAttr(<?php echo js_escape($str_q8); ?>));
+    question.appendChild(text);
+    var new_line = document.createElement("br"); // second part is in italics
+    var ital = document.createElement("i"); // second part is in italics
+    var question_2 = document.createTextNode(jsAttr(<?php echo js_escape($str_q8_2); ?>));
+    ital.appendChild(question_2);
+    question.name = "eighth";
+    question.appendChild(new_line);
+    question.appendChild(ital);
+// populate the   the menue
+     menue.options[0] = new Option ( <?php echo js_escape($str_not); ?>, "0");
+     menue.options[1] = new Option ( <?php echo js_escape($str_somewhat); ?>, "1");
+     menue.options[2] = new Option ( <?php echo js_escape($str_very); ?>, "2");
+     menue.options[3] = new Option ( <?php echo js_escape($str_extremely);?>, "3");
+     menue.options[4] = new Option ( <?php echo js_escape($str_default);  ?>, "undef");
+}
+</script>
+<form method=post action="<?php echo $rootdir;?>/forms/gad7/save.php?mode=new" name="my_form" onSubmit="return(check_all());" >
+<input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+<br></br>
+<span><font size=4><?php echo text($str_form_name); ?></font></span>
+<br></br>
+<input type="Submit" value="<?php echo xla('Save Form'); ?>" style="color: #483D8B" >
+ &nbsp &nbsp
+ <input type="button" value="<?php echo text($str_nosave_exit);?>" onclick="top.restoreSession();return( nosave_exit());" style="color: #483D8B">
```

Comment:
attr() instead of text() since within html attributes:
```
<input type="button" value="<?php echo attr($str_nosave_exit);?>" onclick="top.restoreSession();return( nosave_exit());" style="color: #483D8B">
```